src/api/Gemfile

```diff
@@ -41,8 +41,6 @@ gem 'sanitize'
 gem 'pundit'
 # for password hashing
 gem 'bcrypt'
-#
-gem 'responders', '~> 3.0'
```

Comment:
Lets wait until rails-assets.org is back. Since we are removing gem, to make sure that the package is building would be a good thing

Reply:
rails-assets.org is back. Everything is green.